e2e/specs/add-note.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test } from '../core';
+import { expect } from '@playwright/test';
+import { NotesPage } from '../pages';
+import { deletePatient, generateRandomPatient, Patient } from '../commands';
+
+let patient: Patient;
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('should be able to add a note to a patient', async ({ page, api }) => {
+  const notesPage = new NotesPage(page);
+
+  await test.step('When I visit the notes page', async () => {
+    await notesPage.goto(patient.uuid);
+  });
+
+  await test.step('Then I start visit', async () => {
+    await notesPage.startVisit();
+    await expect(notesPage.page.getByText('Active Visit')).toBeVisible();
+  });
+
+  await test.step('And I click Record visit notes', async () => {
+    await notesPage.page.getByText('Record visit notes').click();
+  });
+
+  await test.step('And I fill a note', async () => {
+    await notesPage.page.locator('#visitDateTimePicker').fill('03/07/2023');
```

Comment:
Can we use 'getByLabel' or 'getByRole' functions for querying this (instead of using the id)?

Reply:
Same for line 29,31,33